### PR TITLE
Add wget-spider.sh, clarify licensing

### PR DESCRIPTION
**Pull request by obscurerichard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,3 +159,15 @@ shell$ git clone https://github.com/freezingsaddles/freezing-web.git
   * Add the team IDs for the competition teams and any observer teams (ringer teams) into the production `.env` file
   * Restart the services: `cd /opt/compose && docker-compose up -d`
   * Athletes will get assigned to their correct teams as soon as they join exactly one of the defined competiton teams.
+
+# Legal
+
+This software is a community-driven effort, and as such the contributions are owned by the individual contributors:
+
+ Copyright 2015 Ian Will
```
Comment (obscurerichard) [inline README.md]: @iancw FYI - I'm adding a copyright statement for you for your Freezing Saddles contributions

File: README.md [hunk 2]
```diff
@@ -159,3 +159,15 @@ shell$ git clone https://github.com/freezingsaddles/freezing-web.git
...
+ Copyright 2019 Hans Lillelid
```
Comment (obscurerichard) [inline README.md]: @hozn FYI - I'm adding a copyright statement for you for your Freezing Saddles contributions

File: README.md [hunk 3]
```diff
@@ -159,3 +159,15 @@ shell$ git clone https://github.com/freezingsaddles/freezing-web.git
...
+ Copyright 2020 Jon Renaut
```
Comment (obscurerichard) [inline README.md]: @jrenaut  FYI - I'm adding a copyright statement for you for your Freezing Saddles contributions

File: README.md [hunk 4]
```diff
@@ -159,3 +159,15 @@ shell$ git clone https://github.com/freezingsaddles/freezing-web.git
...
+ Copyright 2020 Merlin Hughes
```
Comment (obscurerichard) [inline README.md]: @merlinorg  FYI - I'm adding a copyright statement for you for your Freezing Saddles contributions

File: README.md (hunk 3 above)
Comment (jrenaut) [reply README.md]: What's the significance of the year? Last contribution date?

File: README.md (hunk 1 above)
Comment (iancw) [reply README.md]: Fine with me, thanks for the attribution!

File: README.md (hunk 3 above)
Comment (obscurerichard) [reply README.md]: yep, last contribution date.

**Pull request merged**